sdk/cosmos/azure-cosmos/azure/cosmos/auth.py

```diff
@@ -27,12 +27,23 @@
 import hmac
 
 import six
+import warnings
 
 from . import http_constants
 
 
 def GetAuthorizationHeader(
```

Comment:
I didn't directly remove the method due to the usages in tests and samples since regardless this would give the user insights into it - waiting for confirmation on plan of action.